app/services/concerns/xccdf/rule_group_relationships.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving RuleGroupRelationships
+  module RuleGroupRelationships
+    extend ActiveSupport::Concern
+    included do
+      def save_rule_group_relationships
+        @op_rules_and_rule_groups = @op_rule_groups + @op_rules
+        @rule_group_relationships = all_rule_group_relationships
+
+        ::RuleGroupRelationship.import!(
+          @rule_group_relationships.select(&:new_record?), ignore: true
+        )
+
+        rgr_base = ::RuleGroupRelationship.left_outer_joins(dependent_rule_group: :benchmark)
```

Comment:
I thought that this would list rules too since I'm using `left_outer_joins`. When I tested this, `::RuleGroupRelationship.left_outer_joins(dependent_rule_group: :benchmark).count` was the same as `::RuleGroupRelationship.all.count`

Reply:
The problem is that `dependent_rule_group` has a `class_name`

Reply:
Do we just want all `RuleGroupRelationship`s in this situation? ProfileRules did this:
```
::ProfileRule.joins(profile: :benchmark)
                              .where('profiles.parent_profile_id' => nil)
```
But `::RuleGroupRelationship.all` might work for this?